docs/installation/prepare/pumpkin.md

```diff
@@ -0,0 +1,258 @@
+    },
+    {
+      "name": "11",
+      "device_name": "gpiochip0",
+      "line_number": 173,
+      "pwm_id": -1
+    },
+    {
+      "name": "13",
+      "device_name": "gpiochip0",
+      "line_number": 152,
+      "pwm_id": -1
+    },
+    {
+      "name": "15",
+      "device_name": "gpiochip0",
+      "line_number": 94,
+      "pwm_id": -1
+    },
+    {
+      "name": "19",
+      "device_name": "gpiochip0",
+      "line_number": 163,
+      "pwm_id": -1
+    },
+    {
+      "name": "21",
+      "device_name": "gpiochip0",
+      "line_number": 161,
+      "pwm_id": -1
+    },
+    {
+      "name": "23",
+      "device_name": "gpiochip0",
+      "line_number": 164,
+      "pwm_id": -1
+    },
+    {
+      "name": "27",
+      "device_name": "gpiochip0",
+      "line_number": 82,
+      "pwm_id": -1
+    },
+    {
+      "name": "29",
+      "device_name": "gpiochip0",
+      "line_number": 98,
+      "pwm_id": -1
+    },
+    {
+      "name": "31",
+      "device_name": "gpiochip0",
+      "line_number": 12,
+      "pwm_id": -1
+    },
+    {
+      "name": "33",
+      "device_name": "gpiochip0",
+      "line_number": 101,
+      "pwm_id": -1
+    },
+    {
+      "name": "35",
+      "device_name": "gpiochip0",
+      "line_number": 171,
+      "pwm_id": -1
+    },
+    {
+      "name": "37",
+      "device_name": "gpiochip0",
+      "line_number": 169,
+      "pwm_id": -1
+    },
+    {
+      "name": "8",
+      "device_name": "gpiochip0",
+      "line_number": 115,
+      "pwm_id": -1
+    },
+    {
+      "name": "10",
+      "device_name": "gpiochip0",
+      "line_number": 121,
+      "pwm_id": -1
+    },
+    {
+      "name": "12",
+      "device_name": "gpiochip0",
+      "line_number": 170,
+      "pwm_id": -1
+    },
+    {
+      "name": "16",
+      "device_name": "gpiochip0",
+      "line_number": 165,
+      "pwm_id": -1
+    },
+    {
+      "name": "18",
+      "device_name": "gpiochip0",
+      "line_number": 1,
+      "pwm_id": -1
+    },
+    {
+      "name": "22",
+      "device_name": "gpiochip0",
+      "line_number": 2,
+      "pwm_id": -1
+    },
+    {
+      "name": "24",
+      "device_name": "gpiochip0",
+      "line_number": 162,
+      "pwm_id": -1
+    },
+    {
+      "name": "26",
+      "device_name": "gpiochip0",
+      "line_number": 0,
+      "pwm_id": -1
+    },
+    {
+      "name": "28",
+      "device_name": "gpiochip0",
+      "line_number": 83,
+      "pwm_id": -1
+    },
+    {
+      "name": "32",
+      "device_name": "gpiochip0",
+      "line_number": 97,
+      "pwm_id": -1
+    },
+    {
+      "name": "36",
+      "device_name": "gpiochip0",
+      "line_number": 151,
+      "pwm_id": -1
+    },
+    {
+      "name": "38",
+      "device_name": "gpiochip0",
+      "line_number": 174,
+      "pwm_id": -1
+    },
+    {
+      "name": "40",
+      "device_name": "gpiochip0",
+      "line_number": 172,
+      "pwm_id": -1
+    }
+  ]
+}
+```
+
+## Configure a `customlinux` board
```

Comment:
Seems like this duplicates a lot of info— could we link to customlinux here?

Reply:
Or maybe move the bulk of the pin mappings stuff over here and take it out of customlinux? Then it can be part of the setup docs specific to the board, and we can just link to here from there as an example (can explore in another PR though I suppose)

Reply:
Let's discuss on slack

Reply:
Good w/ me to keep